core/src/main/java/org/mskcc/cbio/portal/util/DataSetsUtil.java

```diff
@@ -132,14 +132,15 @@ public DataSetsUtil() {
 			int tumorMIRNA = getCount(cancerStudy, "_microrna");
 			int methylationHM27 = getCount(cancerStudy, "_methylation_hm27");
 			int rppa = getCount(cancerStudy, "_rppa");
+			int massSpec = getCount(cancerStudy, "_ms_abundance");
```

Comment:
note, once you update the list of allowed types (according to my first comment) you may need to update this one...